aerogear/native-android-example
# Subscriptions

### Summary

Added subscriptions. 

### To test

1. Make sure Ionic showcase server is running
2. Set server url in `Client.java`
3. Run app on multiple devices
4. `Add` `Delete` task should be reflected on both devices
